src/app/components/Tag/Tag.tsx

```diff
@@ -3,25 +3,28 @@ import type { ReactNode } from 'react';
 import styled from 'styled-components';
 
 type TagProps = {
+  id: number;
   children: ReactNode;
   active?: boolean;
   count?: string;
+  onClick: () => void;
```

Comment:
I would make this optional if you have a use case in which you use Tag without `onClick`:

```suggestion
  onClick?: () => void;
```